docs/organizations/overview.mdx

```diff
@@ -5,26 +5,142 @@ description: Learn about how to create and manage Clerk organizations and their
+
+The easiest way to allow users to switch between their personal account and their organization account is to use Clerk's [`<OrganizationSwitcher />`](/docs/components/organization/organization-switcher) component. Once a user selects an organization, that organization is set as the active organization.
+
+You can also use the `setActive()` method, which is available on the [`Clerk`](/docs/references/javascript/clerk/session-methods#set-active) object and is returned by the [`useOrganizationList()`](/docs/references/react/use-organization-list) hook.
+
+## Create an organization
+
+### Application owner
+
+You can create organizations in the Clerk Dashboard or in your application. To create an organization in the Clerk Dashboard:
+
+1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=organizations).
+2. In the navigation sidebar, select **Organizations**.
+3. Select the **Create Organization** button.
+4. Enter the organization's name and slug. The slug is a unique identifier for the organization and is used in URLs. Select the organization's owner from the list of users in your application. The owner is the user who will be the organization's admin.
+
+The number of monthly active organizations (MAOs) you can create depends on your [Clerk plan](/pricing).
+
+With the Free plan, you can create up to 100 MAOs across your entire account. Each MAO can have up to 5 members. So if you have 20 different Clerk applications, each of them can have 5 organizations (100 total across your account).
+
+With the Pro plan, you can create up to 100 MAOs across your entire account. Each MAO after the first 100 costs $1.00 per month. Each MAO can have an unlimited number of members. So if you have 20 applications, each of them can have 5 organizations (100 total across your account), and then each additional organization costs $1.00 per month.
+
+If you need more organizations, please contact [our sales team](https://clerk.com/contact/sales) to upgrade to the Enterprise plan.
+
+### Application user
+
+Users can create organizations within your application. When a user creates an organization, they become the organization's admin. As the organization's admin, they have full control over the organization, including the ability to update the organization's settings, invite users to join the organization, and manage the organization's members.
+
+A user within one of your applications can create up to 100 organizations in that application. This limit is separate from the number of [organizations *you* can create](#application-owner) across your entire account.
+
+The easiest way to allow users to create organizations is to use Clerk's [`<CreateOrganization />`](/docs/components/organization/create-organization) component. This component provides a form for users to create a new organization.
```

Comment:
Organization switcher also allows creation, and is often the onramp to this ability